linux/deploy_vm/templates/post_install_scripts/freebsd_post_config.sh

```diff
@@ -133,6 +133,10 @@ sed -i '' 's/kern.geom.label.gptid.enable="0"/kern.geom.label.gptid.enable="1"/'
 echo 'kern.geom.label.gpt.enable="1"' >>/boot/loader.conf
```

Comment:
This change is in the other merge request?

Reply:
Yes. When the https://github.com/vmware/ansible-vsphere-gos-validation/pull/662 is merged, rebasing will remove this change from this PR.